docs/Makefile

```
@@ -61,7 +61,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 htmlview: html
-	$(PYTHON) -c "import webbrowser; webbrowser.open('_build/html/index.html')"
+	$(PYTHON) -m webbrowser $(BUILDDIR)/html/index.html
```

I would add quotations.
```suggestion
	$(PYTHON) -m webbrowser "$(BUILDDIR)/html/index.html"
```

---

I tried to be aligned with other commands in the Makefile which don't have quotations.
Maybe we can review the file in another PR and add quotations?

---

I don't think that quotations are needed everywhere. I'd add them here to follow an example from Python docs: https://docs.python.org/3/library/webbrowser.html#module-webbrowser.

---

Ok, make sense. I updated the commit.